### ISSUE TITLE
not need to apply the softmax activation function

### ISSUE THREAD

**Issue by SatCat** [hse-ds/iad-deep-learning#6]
в 
   2020/sem03)/sem03_solution.ipynb
не нужен 
    x = self.softmax(self.fc2(x))
т.к. гуглится такое:
do not need to apply the softmax activation function as the last layer of the model, because the softmax is integrated with the nn.CrossEntropyLoss loss function.
